public_html/wp-content/plugins/pattern-translations/pattern-translations.php

```diff
@@ -32,21 +32,24 @@
  * Creates or updates a localised pattern.
  */
 function create_or_update_translated_pattern( Pattern $pattern ) {
+	$parent = get_post( $pattern->parent->ID );
```

Comment:
This will throw a notice if the parent isn't available, but this function probably shouldn't be called with that input anyway.

Reply:
Ah good call. Changed in 880ea43